src/server/db_slice.cc

```diff
@@ -794,7 +794,8 @@ void DbSlice::FlushDbIndexes(const std::vector<DbIndex>& indexes) {
     std::swap(db_arr_[index]->trans_locks, flush_db_arr[index]->trans_locks);
   }
 
-  CHECK(fetched_items_.empty());
+  LOG_IF(DFATAL, fetched_items_.empty())
```

Comment:
Since now we don't crash in release I will ask for the logs (just to make sure that we don't have any gaps)